projects/plugins/jetpack/extensions/shared/memberships/settings.js

```diff
@@ -85,6 +85,46 @@ function NewsletterAccessSetupNudge( { stripeConnectUrl, isStripeConnected, hasN
 	}
 }
 
+function TierSelector() {
+	// TODO: filter on currency?
+	const products = useSelect( select => select( membershipProductsStore ).getProducts() )
+		.filter(
+			product =>
+				product.subscribe_as_site_subscriber &&
+				product.interval === '1 month' &&
+				product.buyer_can_change_amount === false
```

Comment:
```
product.buyer_can_change_amount === false
```

I understand the idea about that, but let's not worry about that for now..
Meaning if subscribers can change the price of their subscription, it is fine. What counts is the price set by the content creator.